GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -803,6 +789,21 @@ IReadOnlyList<GitItemStatus> GetAllChangedFilesWithSubmodulesStatus()
             }
         }
 
+        private void ConfigureMessageBox()
```

Comment:
```suggestion
        private void ConfigureMessageBox(CommitKind commitKind, bool useFormCommitMessage)
```

This way we can test it. Also please add tests.

Reply:
Pro: It emphasizes the depencies.
The drawback would be that all three callers must specify these options without being able to declare `CommitInfo` `static`. It could still evaluate other members.

I don't see any substantial benefit to add a test for a method like this.
Test cases would be only a guard against the modification of this very simple logic (in addition to the mandatory PR review).
```
       private void ConfigureMessageBox(CommitKind commitKind, bool useFormCommitMessage)
        {
            modifyCommitMessageButton.Visible = useFormCommitMessage && commitKind != CommitKind.Normal;
            bool messageCanBeChanged = useFormCommitMessage && commitKind == CommitKind.Normal;

            Message.Enabled = messageCanBeChanged;

            commitMessageToolStripMenuItem.Enabled = messageCanBeChanged;
            commitTemplatesToolStripMenuItem.Enabled = messageCanBeChanged;

            Message.WatermarkText = useFormCommitMessage
                ? _enterCommitMessageHint.Text
                : _commitMessageDisabled.Text;
        }
```

Reply:
I guess this has become obsolete with the new property `CommitKind`.
Your bad feeling regarding this method was justified, once more. 👍